File: Celsius to Fahrenheit.cpp
#include <iostream>
#include <iomanip> // Include the header for setprecision
using namespace std;

// Function to convert Celsius to Fahrenheit
double celsiusToFahrenheit(double celsius)
{
    return (celsius * 9 / 5) + 32;
}

int main()
{
    // Input temperature in Celsius
    double celsius;
    cin >> celsius;

    // Convert Celsius to Fahrenheit
    double fahrenheit = celsiusToFahrenheit(celsius);

    // Print the result with two decimal points
    cout << "The temperature in Fahrenheit is: " << fixed << setprecision(2) << fahrenheit << endl;

    return 0;
}

File: Celsius to Fahrenheit.py
# Function to convert Celsius to Fahrenheit
def celsius_to_fahrenheit(celsius):
    return (celsius * 9/5) + 32

# Input temperature in Celsius
celsius = float(input())

# Convert Celsius to Fahrenheit
fahrenheit = celsius_to_fahrenheit(celsius)

# Print the result with two decimal points
print("The temperature in Fahrenheit is: {:.2f}".format(fahrenheit))